<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfflinetagsClass</class>
 <widget class="QMainWindow" name="OfflinetagsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Offlinetags</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="widgetImage" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string>MainImage</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widgetOptions" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Image Loader</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Device:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBoxDevice">
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Image Analyzer</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="5" column="3">
              <widget class="QSpinBox" name="spinBoxBrightness1">
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Contrast 2:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxContrast1">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>0.500000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Contrast 1:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Brightness 1:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="checkBoxContrast1">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="4">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Change contrast and brightness?</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBoxBlur">
               <property name="text">
                <string>Blur</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QSpinBox" name="spinBoxBlur">
               <property name="maximum">
                <number>10</number>
               </property>
               <property name="value">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item row="4" column="3">
              <widget class="QCheckBox" name="checkBoxContrast4">
               <property name="text">
                <string>4</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Blur value:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QCheckBox" name="checkBoxContrast3">
               <property name="text">
                <string>3</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="checkBoxContrast2">
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Contrast 3:</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Contrast 4:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Brightness 2:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Brightness 3:</string>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Brightness 4:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxContrast2">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>0.500000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxContrast3">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>0.500000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxContrast4">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="minimum">
                <double>0.500000000000000</double>
               </property>
               <property name="maximum">
                <double>5.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="6" column="3">
              <widget class="QSpinBox" name="spinBoxBrightness2">
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
              </widget>
             </item>
             <item row="7" column="3">
              <widget class="QSpinBox" name="spinBoxBrightness3">
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
              </widget>
             </item>
             <item row="8" column="3">
              <widget class="QSpinBox" name="spinBoxBrightness4">
               <property name="minimum">
                <number>-100</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QCheckBox" name="checkBoxCircle">
               <property name="text">
                <string>Circles</string>
               </property>
              </widget>
             </item>
             <item row="9" column="2">
              <widget class="QCheckBox" name="checkBoxTags">
               <property name="text">
                <string>Tags</string>
               </property>
              </widget>
             </item>
             <item row="9" column="3">
              <widget class="QCheckBox" name="checkBoxResult">
               <property name="text">
                <string>Result</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Intervall:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QSpinBox" name="spinBoxIntervall">
               <property name="minimum">
                <number>1000</number>
               </property>
               <property name="maximum">
                <number>60000</number>
               </property>
               <property name="singleStep">
                <number>1000</number>
               </property>
               <property name="value">
                <number>2000</number>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QCheckBox" name="checkBoxFaces">
               <property name="text">
                <string>Faces</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Circle Detector</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Min distance:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>param1:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>param2:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Min radius:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Max radius:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBoxMinDistance">
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="value">
                <number>15</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBoxParam1">
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="singleStep">
                <number>75</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBoxParam2">
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="singleStep">
                <number>275</number>
               </property>
               <property name="value">
                <number>150</number>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="spinBoxMinRadius">
               <property name="maximum">
                <number>5000</number>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSpinBox" name="spinBoxMaxRadius">
               <property name="maximum">
                <number>5000</number>
               </property>
               <property name="value">
                <number>750</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Tag Detector</string>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>threshold1:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>threshold2:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBoxThreshold1">
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBoxThreshold2">
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="value">
                <number>150</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="checkBoxDillate">
               <property name="text">
                <string>Dillate</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frameMini">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <widget class="QLabel" name="labelMini05">
       <property name="geometry">
        <rect>
         <x>439</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img05</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini03">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img03</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini10">
       <property name="geometry">
        <rect>
         <x>988</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img10</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini07">
       <property name="geometry">
        <rect>
         <x>659</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img07</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini04">
       <property name="geometry">
        <rect>
         <x>329</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img04</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini08">
       <property name="geometry">
        <rect>
         <x>769</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img08</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini01">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img01</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini06">
       <property name="geometry">
        <rect>
         <x>549</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img06</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini09">
       <property name="geometry">
        <rect>
         <x>878</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img09</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelMini02">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>0</y>
         <width>28</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>img02</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionLoad_image">
   <property name="text">
    <string>Load image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="offlinetags.qrc"/>
 </resources>
 <connections/>
</ui>
